host/xtest/regression_1000.c

```diff
@@ -2641,3 +2641,50 @@ static void xtest_tee_test_1035(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 1035, xtest_tee_test_1035, "Test BTI");
 #endif
+
+static void xtest_tee_test_1036(ADBG_Case_t *c)
+{
+	TEEC_Session session = { };
+	TEEC_Operation op = TEEC_OPERATION_INITIALIZER;
+	uint32_t ret_orig = 0;
+	TEEC_Result res = TEEC_SUCCESS;
+
+	op.paramTypes = TEEC_PARAM_TYPES(TEEC_NONE, TEEC_NONE, TEEC_NONE,
+					 TEEC_NONE);
+
+	if (!ADBG_EXPECT_TEEC_SUCCESS(c,
+			xtest_teec_open_session(&session, &os_test_ta_uuid,
+						NULL, &ret_orig)))
+		return;
+
+	Do_ADBG_BeginSubCase(c, "PAuth NOP test");
+
+	res = TEEC_InvokeCommand(&session, TA_OS_TEST_CMD_PAUTH_NOP, &op,
+				 &ret_orig);
+	if (res == TEEC_ERROR_NOT_SUPPORTED) {
+		Do_ADBG_Log("Binary doesn't support PAuth - skip tests");
+		Do_ADBG_EndSubCase(c, "PAuth NOP test");
```

Comment:
Note that this isn't strictly needed, but it doesn't harm either.

Reply:
Thanks. To maintain consistency with the other error for this testcase, will remove Do_ADBG_EndSubCase from here.